src/Post.js

```diff
@@ -0,0 +1,99 @@
+import React, {Component} from 'react';
+import Comment from './Comment.js';
+import './Post.css';
+
+class Post extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: props.id,
+      by: null,
+      title: null,
+      url: null,
+      error: null,
+      kids: null,
+      iconClass: 'fas fa-plus',
+      postClass: 'hidden',
+      hasKids: false
+    }
+
+
+    this.getPostClass = this.getPostClass.bind(this);
```

Comment:
Not that I'd change it but arrow functions are automatically bound to lexical 'this' so in your clicks you can do

() => this.getPostClass()

instead of having to manually bind